packages/big-design/src/components/Worksheet/Row/Row.tsx

```diff
@@ -14,12 +20,20 @@ interface RowProps<Item> {
 const InternalRow = <T extends WorksheetItem>({ columns, rowIndex }: RowProps<T>) => {
   const row = useStore(useMemo(() => (state) => state.rows[rowIndex], [rowIndex]));
 
+  const hasFormatting = useCallback((column: WorksheetColumn<T>): column is
+    | WorksheetTextColumn<T>
+    | WorksheetNumberColumn<T>
+    | WorksheetModalColumn<T> => {
+    return column.type === 'text' || column.type === 'number' || column.type === 'modal';
+  }, []);
+
   return (
     <tr>
       <RowStatus rowIndex={rowIndex} />
       {columns.map((column, columnIndex) => (
         <Cell
           columnIndex={columnIndex}
+          formatting={hasFormatting(column) ? column.formatting : undefined}
```

Comment:
Couldn't we just use optional chaining + nullish coalescing here? Or am I missing something?
```suggestion
          formatting={column?.formatting ?? undefined}
```

Reply:
We can't because `formatting` isn't present in all `column` types.

Reply:
Ah I guess it's TS in this case complaining on the checkbox column types.